react/react1/week2/todo-list/src/components/ToDoList.jsx

```diff
@@ -0,0 +1,85 @@
+import TaskItem from "./TaskItem";
+import tasklist from "../data/tasklist";
+import { useState } from "react";
+
+export default function ToDoList() {
+  const setTaskList = () => {
```

Comment:
Defining variable names like setTaskList is suggested to use only while defining useState, otherwise, the code looks confusing.

Reply:
Instead of `setTaskList`, you can just define like `tasks`.